### PR TITLE
Feature/improve docs

### PR DESCRIPTION
**Pull request by mindywhitsitt**
Now that Protractor-sync is wrapping Protractor instead of extending it, I felt that the API documentation should be more complete. At the same time, it didn't seem worthwhile to spend days writing exhaustive documentation and examples of each method. I hope I've arrived at an acceptable middle ground here.

@scriby I'd appreciate your thoughts if you have the time

### CODE REVIEW

**Comment by shermaneric** [comment]
Changes looking good! Still reading through:
Random/possibly unrelated is that the circleci test failed because it couldn't find the selenium-webdriver module. 

That happened once to me here and I realized that I had a selenium-webdriver node_modules directory somewhere off of my home dir and it was causing problems...

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        var settings = elementSync.findVisible('.settings'); //Finds exactly one visible element with a class of "settings"
+        settings.findVisible('input.start-date').clear().sendKeys('1/1/2000');
+        settings.findVisible('.save').scrollIntoView().click();
+    
+        settings.waitUntilRemoved();
+        
+        polledExpect(function() { return element.findVisible('div.start-date').getText(); }).toEqual('1/1/2000');
+        //With ES6/Typescript: polledExpect(() => element.findVisible('div.start-date').getText()).toEqual('1/1/2000');
+    });
```
Comment (shermaneric) [inline README.md:43]: Nice call splitting up the ES6 differential

Comment (mindywhitsitt) [reply README.md:43]: Yep. Credit to @scriby for that.

**Review by shermaneric — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+See test/spec/protractor-sync_test.ts for more examples.
+
+# How to contribute
+ 
+Thanks you for your interest in Protractor-sync.  In lieu of a formal style guide, take care to maintain the existing coding style. Please add tests for any new or changed functionality.
+
+# API
+
+See API.md
+
```
Comment (casteaux) [inline README.md:57]: Nice!

**Review by casteaux — APPROVED** [review]

File: API.md [hunk 3]
```diff
@@ -203,7 +203,8 @@ Util methods that stand alone as part of Protractor-sync.
 
 Simple example usage:
 ````
-import {polledExpect} from 'protractor-sync';
+import {configure, polledExpect} from 'protractor-sync';
```
Comment (shermaneric) [inline API.md]: It's good to show how to import multiple classes. nice!

**Review by sharp-bits — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -47,17 +49,17 @@ See test/spec/protractor-sync_test.ts for more examples.
 
 # How to contribute
  
-Thanks you for your interest in protractor-sync.  In lieu of a formal styleguide, take care to maintain the existing coding style. Please add tests for any new or changed functionality. 
+Thanks you for your interest in Protractor-sync.  In lieu of a formal style guide, take care to maintain the existing coding style. Please add tests for any new or changed functionality.
```
Comment (shermaneric) [inline README.md]: minor:  I don't know if it's more or less correct to capitalize protractor-sync.  I leave it in your hands :)

**Review by shermaneric — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (mindywhitsitt) [reply README.md]: Ha. I didn't know either! As you can see I changed it to be capitalized. Perhaps you or someone else will change it back later. :)

**Pull request merged**